modules/gapi/src/backends/fluid/gfluidcore_simd_sse41.hpp

```diff
@@ -216,8 +216,8 @@ CV_ALWAYS_INLINE void calcRowLinear_8UC_Impl_<3>(uint8_t* dst[],
                                               const int      lpi) {
     bool xRatioEq = inSz.width == outSz.width;
     bool yRatioEq = inSz.height == outSz.height;
-    constexpr int nlanes = 16;
-    constexpr int half_nlanes = 16 / 2;
+    constexpr int nlanes = 16; // number of 8-bit integers that fit into a 128-bit SIMD vector.
```

Comment:
suggest to make code self documented ( may be in future ways) likes as
```
struct sse_traits {
constexpr int instruction_size = 128;
constexpr int lanes_count = 128 / 8bit;
...
}
```
and reuse them